app/views/users/show.html.erb

```diff
@@ -1 +1,21 @@
-<h1>user profil</h1>
\ No newline at end of file
+<h1>The users profil</h1>
+<div>
+    <div class='user-container'>
+      <%= image_tag @user.photo, :class => 'user-photo', alt: 'User photo' %>
+      <h2><%= @user.name %></h2>
+      <p>Number of posts: <%= @user.PostsCounter.nil? ? 0 : @user.user_posts %></p>
+      <h2>Bio</h2>
+      <p><%= @user.bio %></p>
+      <h1>Posts list</h1>
+      <% @user.user_posts.each do |post| %>
+        <div class='post-card'>
+        <h3><%= post.title %></h3>
+        <p><%= post.text %></p>
+        <div class='likes-comments'>
+          <p>Comments: <%= post.CommentCounter.nil? ? 0 : post.CommentCounter %></p>
+          <p>Likes: <%= post.LikesCounter.nil? ? 0 : post.LikesCounter %></p>
+        </div>
+      </div>
+      <% end %>
+    </div>

```

Comment:
- [ ] In the users_show page, there should be a link `See all the posts` that opens the user's post index as displayed on the [wireframe](https://raw.githubusercontent.com/microverseinc/curriculum-rails/main/blog-app/images/blog_user_all_posts.png?token=ABIFLA3SVPWT6MXQ3C2OGFLBHIHES). Kindly add that.